resources/extensions/document_info_post_processor.rb

```diff
@@ -0,0 +1,38 @@
+require 'asciidoctor/extensions' unless RUBY_ENGINE == 'opal'
+
+include Asciidoctor
+
+class DocumentInfoPostProcessor < Extensions::Postprocessor; use_dsl
+  def process(document, output)
+    if document.backend == 'html5'
+      if (outfile = document.attr 'outfile')
+        require 'yaml'
+        info = {}
+        tags = []
+        if document.attr? 'tags'
+          tags = (document.attr 'tags').split(',').map { |tag| tag.strip! }.reject { |tag| tag.empty? }
+        end
+        tags << 'public' if document.attr? 'public'
```

Comment:
this shouldn't be a tag

Reply:
Currently, tags are not working, see: https://github.com/neo4j-labs/neo4j-labs-pages/pull/4/files#diff-be8f1c033072e78dc1e971188b66cd7cR224-R225

For now, we publish pages without tags because I think the API expect tag ids but I don't know how to get the id for a given tag.

Reply:
In KB we use the public-private to prevent from publishing to WP.

Perhaps we can use this like a "permission to publish" attribute, i.e. if that's configured e.g. `publishPermissionAttribute: 'public'` then it's used to prevent pages without that attribute from being published (safety mechanism).

Hmm, we need to check in the Knowledge Base WP publisher how to deal with WP tags. I think it just creates tags on demand.

Reply:
The public-private publishing is handled by the "stage" property.

> Hmm, we need to check in the Knowledge Base WP publisher how to deal with WP tags. I think it just creates tags on demand.

I found this line of code but I don't know how to translate it using the (new) REST API: https://github.com/cheerfulstoic/ascii_press/blob/66037ef28f36d7b4262be9a10fa06d9240436f07/lib/ascii_press.rb#L204